src/ert/storage/local_ensemble.py

```diff
@@ -295,19 +270,21 @@ def export_field(
             raise KeyError(
                 f"Unable to load FIELD for key: {key}, realization: {realization} "
             )
-        data = np.load(data_path / f"{key}.npy")
+        da = xr.open_dataarray(data_path / f"{key}")
+        # Squeeze to get rid of realization-dimension
+        data: npt.NDArray[np.double] = da.values.squeeze(axis=0)
         data = field_transform(data, transform_name=info["output_transformation"])
         data = _field_truncate(
             data,
             info["truncation_min"],
             info["truncation_max"],
         )
 
-        save_field(
+        field_utils.save_field(
             data,
             key,
             self.experiment.grid_path,
-            Shape(info["nx"], info["ny"], info["nz"]),
+            field_utils.Shape(info["nx"], info["ny"], info["nz"]),
             output_path,
             fformat,
         )
```

Comment:
Should we inline this whole section into `field_config.py`? We do that in surface and gen_kw, and then `LocalEnsemble` does not know field is a type.

Reply:
Ah much better. I've pushed fixes to your suggestions.